### PR TITLE
Vtx mode led indication

### PR DESCRIPTION
**Pull request by morphal**
Adds TBS-like indication of the current band/channel/power.

To enable feature LED_INDICATION_OF_VTX_MODE should be defined

### CODE REVIEW

File: src/src/main.c [hunk 1]
```diff
@@ -65,6 +68,10 @@ void setup(void)
 
   status_leds_init();
 
+#ifndef LED_INDICATION_OF_VTX_MODE
+  resetModeIndication()
```
Comment (JyeSmith) [inline src/src/main.c:72]: Missing ";"

Comment (morphal) [reply src/src/main.c]: indeed, sorry for that. it also should be #ifdef instead. I've pushed a fix for that

**Comment by morphal** [comment]
@JyeSmith is it worth adding additional build target so that github actions would also run on the led indication code?
image

**Comment by JyeSmith** [comment]
Maybe, but it could be better for users to make this default and define for the devs to revert to the current method.  I know when Im testing on the bench its REALLY handy to see when a packet is being received.

**Comment by morphal** [comment]
@JyeSmith I have enabled the indication feature by default.
To disable it just comment the define in commons.h
`#define LED_INDICATION_OF_VTX_MODE`

**Review by cruwaller — APPROVED** [review]

**Comment by morphal** [comment]
thanks for approval @cruwaller
I have no permissions to merge. What is the following procedure? Should I mention someone in a specific way?

**Review by JyeSmith — APPROVED** [review]

**Comment by JyeSmith** [comment]
Thanks and sorry it took so long.

After testing I wonder if there is also a good way to indicate pitmode.  Maybe no power LED blinks, or the red LED blinks?

**Comment by morphal** [comment]
@JyeSmith how do you like red LED fast blinking for pit mode indication? (kinda like ELRS RX wifi AP mode blinking)

**Pull request merged**